src/python/pants/option/config_test.py

```diff
@@ -51,6 +51,13 @@ class ConfigFile:
         add = 0
         remove = 0
         nested = { nested_key = 'foo' }
+
+        [list_merging]
+        list1 = []
+        list2 = [1, 2]
+        list3.add = [3, 4]
+        list4.remove = [5]
+        list5 = [6, 7]
```

Comment:
It would be really good to use independent test fixtures (with helper methods if needed), and avoid adding to monolithic fixtures where possible. Otherwise tests end up too interdependent on one another.

Reply:
Yeah, I may have to leave that for another day though, as it means a bunch of refactoring on this test.